webview/src/plots/components/DragAndDropPlot.tsx

```diff
@@ -0,0 +1,52 @@
+import cx from 'classnames'
+import React from 'react'
+import { PlotsSection } from 'dvc/src/plots/webview/contract'
+import styles from './styles.module.scss'
+import { getMetricVsParamTitle } from './util'
+import { GripIcon } from '../../shared/components/dragDrop/GripIcon'
+import { Icon } from '../../shared/components/Icon'
+import { GraphLine } from '../../shared/components/icons'
+import { useGetPlot } from '../hooks/useGetPlot'
+
+interface DragAndDropPlotProps {
+  plot: string
+  sectionKey: PlotsSection
+}
+
+export const DragAndDropPlot: React.FC<DragAndDropPlotProps> = ({
+  plot,
+  sectionKey
+}) => {
+  const { spec, isTemplatePlot } = useGetPlot(sectionKey, plot)
+
+  let title = spec?.titles.main.normal as unknown as string
+  let subtitle = ''
+
+  if (!isTemplatePlot) {
+    const yTitle = spec?.titles.y.normal as unknown as string
```

Comment:
[F] We'll be able to get these for all plots with the change in #4734 (more conflicts 😭)